### PR TITLE
[wip] feature: python 3

### PR DESCRIPTION
**Pull request by lsh-0**
* ~upgraded dependencies~
* suffers same problem as bot-lax on upgrade to python3 regarding `requests-cache` (needs to be recreated)

### CODE REVIEW

**Comment by seanwiseman** [comment]
The changes look good to me so far.

File: src/metrics/ga_metrics/bulk.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 __description__ = """Bulk loading of eLife metrics from Google Analytics."""
 
 import os
-import core
-import utils
+from . import core
+from . import utils
+from ..utils import lmap, lfilter
```
[the remark this answers is not in the document]
Comment (lsh-0) [reply src/metrics/ga_metrics/bulk.py]: yes, it should. good catch.  I don't mind the `from . import ...` relative import, but the `from ..foo import ...` remind me of the early php days and dealing with nested relative includes. I don't think python would suffer it's problems though.

**Comment by lsh-0** [comment]
python3 added to environment, gave alfred a kick

**Comment by lsh-0** [comment]
new errors are a result of requests_cache. flushing requests cache on ci

**Pull request merged**